### ISSUE TITLE
[Bug]: Commands do not work in Windows

### ISSUE THREAD

**Issue by RaphaelDDL** [nuxtrdev/nuxtr-vscode#105]
### Environment

Windows10 Pro 22H2
------------------------------
- Operating System: Windows_NT
- Node Version:     v18.16.0
- Nuxt Version:     -
- CLI Version:      3.10.0
- Nitro Version:    -
- Package Manager:  unknown
- Builder:          -
- User Config:      -
- Runtime Modules:  -
- Build Modules:    -
------------------------------

### What went wrong?

- Nuxtr: Create new Nuxt Project

![image](https://github.com/nuxtrdev/nuxtr-vscode/assets/1087109/bf864419-c6e8-4292-a0f2-535c61a67693)

### How to reproduce it?

On Windows:

- Install Node
- Install VSCode
- Install Nuxtr
- Open Settings, filter by `@ext:Nuxtr.nuxtr-vscode`, select NPM as default package manager
- Ctrl+Shift+P, pick `Nuxtr: Create new Nuxt Project`

Get error above. 
Couldn't find any issue opened regarding Win, nor this error. Restarting VSCode didn't make any difference.

**Comment by HybridEyeGrillage** [comment]
I had the same issue as you did, @RaphaelDDL.
Did you modify any of the Nuxtr settings in VSCode? I had quite a few and it seems that by resetting all of then, I managed to be able to do the commands again.

I hadn't done a full investigation into which of the settings is causing the issue, but maybe someone else can look into it.

**Comment by adhamfarrag** [comment]
Hi @RaphaelDDL @HybridEyeGrillage, This bug is caused by activation issues, unrelated to your usage/settings. 

Nuxtr activates only when you have a Nuxt project. If you create a new project when Nuxtr is activated (let's say Nuxt icon in the sidebar is visible), it should work properly. Also, if you notice `Nuxtr: create new project` is the only command you can see when Nuxtr is deactivated, on purpose to be able to create a new project without this issue. 

Apologies for the unfinished work here, have to refactor couple things to be able to make it work without this bug and working on it.

**Comment by HybridEyeGrillage** [comment]
That seems strange still. Yes, I activated Nuxtr on an existing Nuxt project, but after running a few commands, I modified my VSCode settings. After that, I couldn't get to any other commands than `Nuxtr: Create error.vue layout` and `Nuxtr: Create new Project`.

But thanks for confirming it's a known issue.

**Comment by adhamfarrag** [comment]
@HybridEyeGrillage Can you help me reproduce this? What settings did you change ?

**Comment by RaphaelDDL** [comment]
In my case, I installed the plugin and tried to create a new Nuxt project, that is all. Seeing it wasn't working, then I went on options and picked NPM just to make sure.

I assumed that create project would start a new project on an empty folder (kinda like a helping hand for nuxi command) as readme says "Using Nuxtr: Create Nuxt project command, you can create a new Nuxt project with a few steps". Am I mistaken? You said "Nuxtr activates only when you have a Nuxt project."... so I need a existing project to create a project? o_O

**Comment by HybridEyeGrillage** [comment]
> @HybridEyeGrillage Can you help me reproduce this? What settings did you change ?

Let's see...
```txt
* Default Package Manager: pnpm
* Open files after creation [x]
* Pinia Files: 
  *	Default Template: setup
* Vue Files: 
  *	First Tag: script
  * Always Scoped [X]
  * Default Language: scss
* Editor:
  * Default Folding Range Provider: Volar 
```
After doing this, I reloaded the window and then tried to get the Nuxtr commands; I only saw the two options that I mentioned before again.
I also didn't import any other custom settings (no `.vscode` folder or `setings.json` file), so it's the default VSCode settings. Not sure using WSL: Ubuntu also plays into this... 
I hope that this doesn't detracts from OP's initial post, but the error is the same.

**Comment by adhamfarrag** [comment]
@HybridEyeGrillage, Currenly Nuxtr only supports having one FE layer in your monorepo, and you should [configure it manually](https://github.com/nuxtrdev/nuxtr-vscode#support-for-srcdir-serverdir-and-monorepo-projects) using `  "nuxtr.monorepoMode.DirectoryName": "directory-name". `

Now I believe that @RaphaelDDL's issue is different from yours. Can you open a new issue if my suggestion didn't work for you ?

**Comment by adhamfarrag** [comment]
@RaphaelDDL Nuxtr 0.2.13 should include fixes for your bug. Please, update.

**Comment by RaphaelDDL** [comment]
Updated, same error. So I uninstalled, reloaded, reinstalled, reloaded, `v0.2.13` active, same error.

So I played around a bit more and I found out the difference between work vs doesn't work:  

- If you open VSCode window, but has no folder open (`open folder...` option), I get the error of the command not found.
- If I do open a folder, even if is empty and has nothing inside, or has an existing project (doesn't matter, as long as a folder was open), then when trying the command, it works and asks me which type I want (nuxt, nuxt with ui, content, module, layer), asks a project name then prompts me where the project folder should be created and lastly, if I want to open in existing window or new window (both work).

So yeah, basically the command does not work if you only have a VSCode window open, clean, no folder. Whenever opening any folder whatsoever, then it works in all steps. Opening only a File does not work either.
